metashare/api/serializers.py

```diff
@@ -29,3 +31,36 @@ class Meta:
             "slug",
             "old_slugs",
         )
+
+
+class ProjectSerializer(serializers.ModelSerializer):
+    id = serializers.CharField(read_only=True)
+    description = MarkdownField(allow_blank=True)
+    product = serializers.PrimaryKeyRelatedField(
+        queryset=Product.objects.all(), pk_field=serializers.CharField()
+    )
+    branch_url = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Project
+        fields = (
+            "id",
+            "name",
+            "description",
+            "slug",
+            "old_slugs",
+            "product",
+            "branch_url",
+        )
+
+    def get_branch_url(self, obj):
+        return f"{obj.product.repo_url}/tree/{obj.branch_name}"
+
+    def create(self, validated_data):
+        instance = super().create(validated_data)
+        gh.create_branch(
+            self.context["request"].user,
+            instance.product.repo_url,
+            instance.branch_name,
+        )
+        return instance
```

Comment:
@wlonk Given that we now want to wait on branch creation until the project has a task with a scratch org... should we comment out this block?

Reply:
Yeah, we should.